test/joint_tours/configs/initialize_landuse.yaml

```diff
@@ -0,0 +1,54 @@
+annotate_tables:
+  - tablename: land_use
+    column_map:
+        MAZ_ORIGINAL: zone_id
+        CountyID: county_id
+        DistID: DISTRICT
+        HH: TOTHH
+        POP: TOTPOP
+        ACRES: TOTACRE
+        emp_total: TOTEMP
+    annotate:
+      SPEC: annotate_landuse
+      DF: land_use
+  - tablename: accessibility
```

Comment:
Am I understanding this correctly that the accessibilities used in the jtfc model is not computed by ActivitySim and must be read in via the landuse file?  Is it possible to calculate these accessibilities using ActivitySim's current accessibilities module?

Reply:
We are not running the entire pipeline in our test. We are using accessibilities calculated by CTRAMP as an input to test jtfc without running accessibilities in ActivitySim. When jtfc is run normally in the ActivitySim pipeline, this setting is not needed and it will use the accessibility table calculated in the accessibilities module.